app/services/invite_schools.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class InviteSchools
+  def run(school_ids)
+    logger = Rails.logger
+    logger.info "Emailing schools"
+
+    school_ids.each do |school_id|
+      school = School.find_by(id: school_id)
+
+      if school.nil?
+        logger.info "School not found: #{school_id} ... skipping"
+        next
+      end
+
+      token = reference
+      recipient = recipient_email(school)
```

Comment:
What about if this is blank?

Reply:
It will fail as we need a recipient address to send the email. I think I've covered this in the specs